Fix python version in dockerfile

## Description
Running "docker build ." with current Dockerfile fails because of headers problem, see in error message.
I think the problem is in python image version, which uses latest (therefore 3.8 for now) instead of python 3.6 as stated in the [README](https://github.com/awslabs/gluon-ts/blob/master/README.md).
I fixed this by changing first line of Dockerfile to "FROM python:3.6.5" and it works as expected.
_(Note: didn't try Dockerfile.gpu but it might work since it uses python3.7 explicitly.)_

## To Reproduce
1. git clone https://github.com/awslabs/gluon-ts.git
2. cd gluon-ts
3. git checkout v0.4.2 
3. docker build .

## Error Message
```
gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -Iast27/Include -I/usr/local/include/python3.8 -c ast27/Parser/acceler.c -o build/temp.linux-x86_64-3.8/ast27/Parser/acceler.o
    ast27/Parser/acceler.c:13:10: fatal error: pgenheaders.h: No such file or directory
     #include "pgenheaders.h"
              ^~~~~~~~~~~~~~~
    compilation terminated.
    error: command 'gcc' failed with exit status 1
```
Full error log: [gluon-docker-err.log](https://github.com/awslabs/gluon-ts/files/4023832/gluon-docker-err.log)

## Environment
- Operating system: Ubuntu 18 on x86_64
- Python version: 3.8
- GluonTS version: 0.4.2
- Docker version:  18.09.7, build 2d0083d

p.s.1 Sorry for any mistakes, this is my first issue on GitHub open source project :)
p.s.2 Hope I can also fix this issue (once approved it's a real problem?) with a pull request later.
p.s.3 You did a great job with this project so thanks!

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM python:3
+FROM python:3.7
 
 ADD . /gluonts
 

